tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -575,25 +574,44 @@ user uploads subject
     user chooses file    id:dataFileUploadForm-metadataFile    ${FOLDER}${META_FILE}
     user clicks button    Upload data files
     user waits until h2 is visible    Uploaded data files    %{WAIT_LONG}
-    user waits until page contains accordion section    ${SUBJECT_NAME}    %{WAIT_SMALL}
-    user scrolls to accordion section    ${SUBJECT_NAME}
-    user opens accordion section    ${SUBJECT_NAME}
-    ${section}=    user gets accordion section content element    ${SUBJECT_NAME}
+    user waits until page contains element    testid:Data files table
 
     IF    "${IMPORT_STATUS}" != "Importing"
         user waits until page finishes loading
     END
 
-    user checks headed table body row contains    Status    ${IMPORT_STATUS}    ${section}    %{WAIT_DATA_FILE_IMPORT}
+    user waits until page contains data uploads table
+    ${row}=    user gets table row    row_cell_text=${SUBJECT_NAME}    parent=testid:Data files table
+    ...    wait=%{WAIT_SMALL}
+    ${status_cell}=    user gets testid element    id=Status    wait=%{WAIT_SMALL}    parent=${row}
+    user scrolls to element    ${status_cell}
+    user waits until element contains    element=${status_cell}    text=${IMPORT_STATUS}
+    ...    wait=%{WAIT_DATA_FILE_IMPORT}
```

Comment:
Similar to earlier comment - would switch back to using positional arguments for required parameters.